gen/unmarshal.go

```diff
@@ -325,9 +355,22 @@ func (u *unmarshalGen) gMap(m *Map) {
 	u.msgs = append(u.msgs, resizemsgs...)
 
 	// loop and get key,value
+	last := randIdent()
+	lastSet := randIdent()
+	u.p.printf("\nvar %s %s; _ = %s", last, m.Key.TypeName(), last) // we might not use the sort if it's not defined
```

Comment:
cannot find generated example for this code. I see [StepLess usage](https://github.com/algorand/go-algorand/pull/5605/files#diff-60c8ef764e8f15e0c54e585187cd6c5986b8b7fb69ba4d6bc8d3d23d9ca1d22cR4168) but no the `; _ =` pattern, is the generated code up-to-date?